Add the Enterprise features documentation to the global doc set (#164)
Avaiga/taipy-doc: docs/manuals/gui/binding.md

    - If you have declared `pi = 3.141592653597`, then `<|Pi is {pi:.4f}|>` will
      display the text:<br/>
      `Pi is 3.1416`.
    - If you have `v = 64177`, then `<|dec:{v}, oct:{v:08o}, hex:{v:X}|>` will result
      in displaying the text:<br/>
      `dec:64177, oct:00175261, hex:FAB1`.

    Note that since HTML text eliminates non-significant white space, the
    right-justification format (`{string:>n}`) does not impact the resulting
    display.

## Scope for variable binding

In the tiny example above, the entire application holds a single page.
The bound variable (*x*) is located in the same, unique module.

In larger applications, we may want to create Python modules that hold
Taipy pages, binding visual elements to local variables. That allows for
a clearer organization of the code, where application variables used in
only a few pages can sit next to the page definition.

When Taipy finds a variable in a page, it first tries to locate it in the module
where this page is defined. If the variable can not be found in the page module,
then the variable is sought in the *\_\_main\_\_* module (typically, where the
`Gui^` instance is created).

> This mechanisms allows to create pages on their own module, with local variables

Reviewer: ```suggestion
This mechanisms allows pages to have its own local variables encapsulated within its own module
```

Author: Rephrased.

that the rest of the application may not be aware of. Global variables (the ones
declared in the *\_\_main\_\_* module), on the other hand, can be used in all
page content definitions. The module where the page is defined does not need to
import the global variables, if they are not used in the Python code of the module.

!!! example "Example"

    Say you want to create a page that represents some evalution of an
    expression using a global variable as well as some local parameter.

    This local parameter is useless for the rest of the application, so

>     we store it in a local variable, in the module where the page is

Reviewer: ```suggestion
    we store it as a local variable, in the module where the page is
```

Author: Ok

    defined.<br/>
    We create a _pages_ package, where we can store the file `page.py` which is
    the module file where the page would be declared:

    ```py linenums="1"
    # File: pages/page.py
    from taipy.gui import Markdown

    page = Markdown("""# Expression evaluation
    Expression value: <|{base_value + local_value}|>
    """)

    local_value=0
    ```

    Note that the page definition uses the variable _base_value_ (in line 5) that is
    unknown to this module.

    The page also references the variable _local_value_ (line 8), which is local to
    this module.

    Now let's create the whole application, adding a home page, and reusing
    the page that we just created above. That would be done in the file `main.py`
    defining the *\_\_main\_\_* module of our application:

    ```py  linenums="1"
    # File: main.py
    from taipy.gui import Gui, Markdown
    from pages.page import page

    navigation = [("/home", "Home"), ("/page", "Expression")]

>     root_page="""# Expression

Reviewer 2: find another title ?

Author: Title dropped.

    <|navbar|lov={navigation}|>
    """

>     root_page="""# Home page

Reviewer 2: I suppose this is home_page instead of root_page ?

Author: Fixed. Thanks.
